app/Resources/views/index.html.twig

```diff
@@ -7,6 +7,40 @@
 
 {% block body %}
 
+  {% if not app.user and not isWebview() %}
+    <div id="welcomeSection">
+      <h3>{{ "programs.welcome.title"|trans({}, "catroweb") }}</h3>
+      <h3>{{ "programs.welcome.subtitle"|trans({}, "catroweb") }}</h3>
+      <div class="row">
+        <div class="col-md-8">
+          <img src="{{ asset('images/help/starters.png') }}" height="200px" width="33%" onclick="self.location.href='{{ url('catrobat_web_starter') }}'" />
+          <img src="{{ asset('images/help/tutorials.png') }}" height="200px" width="33%" onclick="self.location.href='{{ url('catrobat_web_tutorialcards') }}'" />
+          <img src="{{ asset('images/help/step_by_step_mobile.png') }}" height="200px" width="33%" onclick="self.location.href='{{ url('catrobat_web_stepByStep') }}'" />
+        </div>
+        <div class="col-md-4">
+            <iframe id="youtube-help-video" height="200px" width="100%" src="https://www.youtube.com/embed/BHe2r2WU-T8" frameborder="0" allowfullscreen></iframe>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-md-6 d-block inline">
```

Comment:
The classes `d-block` and `inline` don't exist inside our css and as far as I could find neither do they in any js library. You can savely remove them if you don't know the purpose of them.

Reply:
This holds for any occurences of `d-block` and `inline` classes inside this PR